libs/backend/src/cast_vote_records/export.ts

```diff
@@ -278,14 +283,22 @@ function buildCastVoteRecord(
     (canonicalizedSheet.type === 'bmd' &&
       canonicalizedSheet.interpretation.ballotId) ||
     unsafeParse(BallotIdSchema, id);
-  const [frontImageFilePath, backImageFilePath] = canonicalizedSheet.filenames;
-  const imageFileUris: SheetOf<string> | undefined =
-    options.shouldIncludeImageReferences
-      ? [
-          `file:${path.basename(frontImageFilePath)}`,
-          `file:${path.basename(backImageFilePath)}`,
-        ]
-      : undefined;
+  const images: SheetOf<CvrImageDataInput> | undefined = referencedFiles
+    ? [
+        {
+          imageHash: await referencedFiles.imageFiles[0].computeSha256Hash(),
+          imageRelativePath: referencedFiles.imageFiles[0].fileName,
+          layoutFileHash:
+            await referencedFiles.layoutFiles?.[0].computeSha256Hash(),
+        },
+        {
+          imageHash: await referencedFiles.imageFiles[1].computeSha256Hash(),
+          imageRelativePath: referencedFiles.imageFiles[1].fileName,
+          layoutFileHash:
+            await referencedFiles.layoutFiles?.[1].computeSha256Hash(),
+        },
+      ]
```

Comment:
Seeing this sort of thing prompted me to create #4059.

Reply:
Ended up using the new and improved `mapSheet` at this very code point, too, haha: https://github.com/votingworks/vxsuite/pull/4056/commits/d2cf1cebee0df2a2be2039a69d95a001eed1a043